docs/cli/dist-git/init.md

```diff
@@ -0,0 +1,89 @@
+---
+title: init
+sidebar_position: 1
+---
+
+# `packit dist-git init`
+
+Create a Packit's dist-git configuration for release syncing to Fedora.
+
+
+## Help
+    
+    $ packit dist-git init
```

Comment:
I guess I could've created a card to automate this thing somehow, I think that I had to fix some inconsistencies a long time ago…

Reply:
good idea :+1:

Reply:
created #825